packages/hoppscotch-app/components/app/PaneLayout.vue

```diff
@@ -56,5 +61,73 @@ const SIDEBAR = useSetting("SIDEBAR")
 
 const slots = useSlots()
 
+let panelVerticalInfo: any = [
+  {
+    max: 100,
+    min: 0,
+    size: 75,
+  },
+  {
+    max: 100,
+    min: 0,
+    size: 65,
+  },
+]
+
+let panelHorizontalInfo: any = [
+  {
+    max: 100,
+    min: 0,
+    size: 75,
+  },
+  {
+    max: 100,
+    min: 0,
+    size: 65,
+  },
+]
+
+function setVerticalPanelInfo(event: any) {
+  // eslint-disable-next-line no-restricted-globals
+  localStorage.setItem("vertical_panel_info", JSON.stringify(event))
```

Comment:
Don't use `locaStorage`. Follow the ESLint error where we mentioned to use `localpersistance.ts` function.